website/docs/tutorials/06-deploying/02-migrations/README.md

```diff
@@ -5,21 +5,18 @@ sidebar_label: Migrations
 # Migrations
 
 :::warning
-This feature is actively developed. Many new additions or changes will land in future Protostar releases.
+Breaking changes can be introduced without depreciation.
 :::
 
 
-Migrations are Cairo files that help you manage contracts on the StarkNet. They are especially useful, when your project consists of multiple contracts. Migration code is mainly written in hints. Protostar injects special functions into hints' scope, which you can use to interact with the StarkNet. These functions are similar to cheatcodes used to test contracts, hence in this document we will refer to these functions as migration cheatcodes.
-
-:::info
-Protostar aims to make migration cheatcodes a subset of [testing cheatcodes](/docs/tutorials/testing/cheatcodes) in order to allow testing migration scripts against Protostar's local StarkNet.
-:::
+Migrations are Cairo files that help you manage contracts on the StarkNet.
```

Comment:
I am missing the big picture here (what really migrations are, where do they work etc.).

Reply:
Done